components/tabs/src/storage.rs

```diff
@@ -443,12 +442,14 @@ mod tests {
 
     #[test]
     fn test_tabs_meta() {
-        let mut db = TabsStorage::new_with_mem_path("test");
+        // Does not create the db on new
```

Comment:
I don't understand this test change?

Reply:
The test before didn't catch that when we call `put_meta` on an empty DB -- the DB is not created. The test would've failed on `get_meta` due to no value (since no DB).

When I try to use `new` vs `new_with_mem_path` it expectantly fails before the new patch in `put_meta`. Even though both of them should've NOT created the db, i believe `new_with_mem_path` still might be creating the DB in mem.

I'll see if I can demonstrate this a little better in one of the tests

Reply:
Running the test:
```
let mut db = TabsStorage::new_with_mem_path("test");
assert!(db.open_if_exists().unwrap().is_none());
```
Fails with:
```
thread 'storage::tests::test_tabs_meta' panicked at 'assertion failed: db.open_if_exists().unwrap().is_none()', components/tabs/src/storage.rs:452:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

so I wanted to adjust the test to ensure that the any writes to the tabs DB, also creates it